catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -117,7 +119,7 @@ public ContentDirectoryMonitor(
       int delayBetweenRetries,
       Executor configurationExecutor,
       Security security) {
-    this.camelContext = camelContext;
+    this.camelContext = Objects.requireNonNull(camelContext, "CamelContext cannot be null");
```

Comment:
✏️ The string should contain the name of variable with the same case. So, it would be `camelContext`, not `CamelContext`.